packages/webapp/src/elections/api/fixtures.ts

```diff
@@ -46,14 +46,14 @@ export const fixtureRegistrationElection: CurrentElection = {
 // In the grand scheme of other fixture data,
 // this would represent the Round 1 election, where there were 5 people in 2 groups
 export const fixtureCurrentElection: CurrentElection = {
-    electionState: "active",
+    electionState: "current_election_state_active",
```

Comment:
We have an enum now for this: `ElectionStatus.Active`

Reply:
awesome, but in the fixture we want to keep the plain value right?

Reply:
ah nvm, I see we adopted the enum for the registration. changing.